qutebrowser/completion/models/sortfilter.py

```diff
@@ -137,12 +137,16 @@ def filterAcceptsRow(self, row, parent):
             # No entries in parent model
             return False
         data = self.srcmodel.data(idx)
-        # TODO more sophisticated filtering
         if not self.pattern:
             return True
-        if not data:
-            return False
-        return self.pattern.casefold() in data.casefold()
+
+        pattern = self.pattern.casefold()
```

Comment:
With a custom filter, should `casefold()` really be called on the pattern already? That seems unexpected, I'd say that's rather the custom filter's responsibility if it wants to do that.